pkg/store/sandbox/metadata.go

```diff
@@ -51,8 +51,8 @@ type Metadata struct {
 	CreatedAt int64
 	// Pid is the process id of the sandbox.
 	Pid uint32
-	// NetNS is the network namespace used by the sandbox.
-	NetNS string
+	// NetNSPath is the network namespace used by the sandbox.
+	NetNSPath string
```

Comment:
We still need to use and initialize this field. The reason why we have this here is that cri-containerd may die.

After cri-containerd comes up, you won't know what network namespace a sandbox is using, thus you won't be able to load and recover it.

The sandbox metadata is checkpointed, so we could read network namespace path from the checkpoint, and reload the network namespace.